mqtt 3/1.0.0/modules/ROOT/pages/edits/mqtt3-connector-quality-of-service.adoc

```diff
@@ -0,0 +1,25 @@
+= Configure the Quality of Service for MQTT 3 Connector
+
+The MQTT protocol enables you to specify levels of delivery assurance for inbound or outbound messages. These levels
+are referred to as quality of service (QoS).
+
+The available QoS levels to configure for MQTT 3 Connector are:
+
+ * *AT_MOST_ONCE*
++
+The message will be delivered at most one time (fire and forget). The level does not guarantee that the message will be received at all.
+ * *AT_LEAST_ONCE*
++
+The message will be delivered correctly at least one time. The message could be delivered more than once.
+ * *EXACTLY_ONCE*
++
+The level guarantees that the message will be delivered correctly exactly one time.
+
+The quality of service can be different for publishing and subscribing, even if the published message and the subscription
```

Comment:
To avoid the modal "can be," per style ref.
```suggestion
The quality of service is sometimes different between message publishing and message subscribing, even if the published message and the subscription
```